### DISCUSSION TITLE
Clover ISO won't boot in VirtualBox

### DISCUSSION THREAD

**Discussion by scorpius** [CloverHackyColor/CloverBootloader#523] [General]
I always get "_A critical error has occurred while running the virtual machine and the machine execution has been stopped._"

Is there a specific setting that I should use? And I need to do this in BIOS mode. I already know it boots just fine in EFI mode.

**Comment by abaumgaertner** [comment]
See this comment: https://github.com/CloverHackyColor/CloverBootloader/issues/559#issuecomment-1308258408
